### PR TITLE
0.0.4

### PR DESCRIPTION
**Pull request by arekkas**
This patch addresses:

* [x] Resolve all issues with redo and text editing #127
* [x] Resolve issues with jumpy focus
* [x] Basic docs for installation, plugins
* [x] Resolve issues with focus and blurring of slate
* [x] Introduce cell jumping using arrow keys
* [x] Fix undo/redo
* [x] empty cell #233 @inyono

### CODE REVIEW

File: docs/walkthrough/content-plugins.md [hunk 1]
```diff
@@ -1,4 +1,62 @@
...
+1. A plugin for [text editing](https://github.com/ory-am/editor/tree/master/src/editor/plugins/content/slate),
+based on [Slate](slatejs.org).
+2. A plugin for [embedding images](https://github.com/ory-am/editor/tree/master/src/editor/plugins/content/image).
+3. A plugin for [embedding videos](https://github.com/ory-am/editor/tree/master/src/editor/plugins/content/video) from Vimeo and YouTube.
+4. A [spacer plugin](https://github.com/ory-am/editor/tree/master/src/editor/plugins/content/spacer).
+
+We encourage you to check out the image, video and spacer plugins!
+Of course, you are not limited to this functionality and can easily write
+your own plugins. Plugins have two parts, one plugin definition and a
+ReactJS component. A minimal plugin definition looks as followed
```
Comment (kevgo) [inline docs/walkthrough/content-plugins.md]: looks as follow~~ed~~__s__:

File: src/editor/components/Controls/Button/index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const Button = ({ description, icon, onClick, active, disabled }: {
   description: string,
   icon: any,
   active: boolean,
-  disabled: boolean,
+  disabled?: boolean,
```
Comment (kevgo) [inline src/editor/components/Controls/Button/index.js]: Does the `?` do anything here?

File: docs/walkthrough/content-plugins.md (hunk 1 above)
Comment (arekkas) [inline docs/walkthrough/content-plugins.md]: Thanks!

File: src/editor/components/Controls/Button/index.js (hunk 2 above)
Comment (arekkas) [inline src/editor/components/Controls/Button/index.js]: Yes, in flow this means that the disabled property can be undefined (is optional). `disabled: ?boolean` would mean that disabled can be null, true or false

**Comment by arekkas** [comment]
This pr can be merged once https://github.com/ianstormtaylor/slate/pull/418 lands

**Comment by arekkas** [comment]
With this pr and #257 I made much more progress than anticipated. A lot of bugs have been resolved in this repo and in dependencies as well. Now a small undo inconsitency has to be checked out and we have to wait until maintainers accept our PRs. Let's hope this happens sometime this week.

**Comment by arekkas** [comment]
Still waiting for https://github.com/omnidan/redux-undo/pull/133 though.

**Pull request merged**